database/migrations/2021_05_05_012647_create_province_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateProvinceTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('province', function (Blueprint $table) {
+            $table->increments('province_id');
+            $table->integer('area_id');
+            $table->string('province_name');
+            $table->timestamps();
+        });
+    }
```

Comment:
Pls do it the same by follow Laravel convention:
Table will be `provinces`
- province_name -> `name`
- province_id -> `id`

Reply:
I change it again.